cpp/open3d/core/Tensor.h

```diff
@@ -225,6 +225,9 @@ class Tensor : public IsDevice {
         return *this;
     }
 
+    /// Tensor reinterpret cast operator.
```

Comment:
- Descirbe what does reinterpret cast do for tensor.
- Describe that the size should match.

Reply:
Done.